eldritchmush/commands/combat.py

```diff
@@ -257,7 +257,12 @@ def updateArmorValue(self, shieldValue, armor, tough, armorSpecialist):
 
         return armor_value
 
-
+	def dmgPenalty(self):
```

Comment:
Did you find where each of these functions are used in the combat and other commands? I need to figure out how this affects those.